deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/cms/controller-cm.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jumpstarter-controller
+  namespace: {{ default .Release.Namespace .Values.namespace }}
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: jumpstarter-controller
+    {{ if .Values.global.timestamp }}
+    deployment.timestamp: {{ .Values.global.timestamp | quote }}
+    {{ end }}
+data:
+  authentication: |
+    apiVersion: jumpstarter.dev/v1alpha1
+    kind: AuthenticationConfiguration
+    # jwt:
```

Comment:
@NickCao can you write some docs on this config, and add the helm values to be able to inject this into place?

May be just a value that allows to pum in the whole authentication configuration?

Reply:
I'm on it.

Reply:
See https://github.com/jumpstarter-dev/jumpstarter-controller/pull/85